src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { UserSubmenuProps } from './types';
+import { userSubmenuItems } from './consts';
+import Icon from '~shared/Icon/Icon';
+
+const displayName = 'common.Extensions.UserNavigation.partials.UserSubmenu';
+
+const UserSubmenu: FC<UserSubmenuProps> = ({ submenuId }) => (
+  <ul className="mt-2">
+    {userSubmenuItems[submenuId].map((item) => (
+      <li key={item.id} className="mb-4 last:mb-0">
+        <Link to={item.url} className="flex items-center">
```

Comment:
Shouldn't use src/components/shared/Extensions/NavLink/NavLink.tsx here?

Reply:
Yeah, i forgot about that component